apps/modal/src/models/download.py

```
@@ -0,0 +1,65 @@
+        from huggingface_hub import snapshot_download
+        print("Downloading F5-TTS details...")
+        # Explicitly set cache_dir to ensure it goes to the volume
+        cache_path = os.path.join(MODEL_DIR, "huggingface") 
+        snapshot_download(
+            repo_id="SWivid/F5-TTS", 
+            allow_patterns=["*.pt", "*.safetensors", "*.generated_files"], # Extended patterns
+            token=os.environ["HF_TOKEN"],
+            cache_dir=cache_path
+        )
+        print(f"F5-TTS models downloaded to {cache_path}")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)